scripts/deploy/localstack/startContainer.sh

```diff
@@ -14,9 +14,9 @@
 set -e
 THIS_DIR=$(cd "$(dirname "$0")" && pwd)
 docker-compose -f "$THIS_DIR/docker-compose.yml" up -d 
-echo "Running localstack container on port 4566\n"
+echo "Running localstack container on port 4566"
 echo "To use sleeper with this container, set the AWS_ENDPOINT_URL environment variable:"
-echo "export AWS_ENDPOINT_URL=http://localhost:4566"
+echo "export AWS_ENDPOINT_URL=http://host.docker.internal:4566"
```

Comment:
It would be nice if we could use localhost either inside or outside the deployment Docker image. This could be done with `--network=host` in the `docker run` commands in `scripts/cli/runInDocker.sh`.